pkg/app/app.go

```diff
@@ -19,7 +19,10 @@ import (
 
 // initConfiguration initializes the configuration with initial values.
 func initConfiguration(config configuration.Configuration, apiClient api.ApiClient, logger *log.Logger) {
-	dir, _ := utils.SnykCacheDir()
+	dir, err := utils.SnykCacheDir()
+	if err != nil {
```

Comment:
nice :)

Reply:
We are already trying to create it inside SnykCacheDir(). I'm careful with exiting since it might be a too aggressive approach and might break things unexpectedly. I think step one is logging to increase transparency.